SQLite3/mORMot.pas

```diff
@@ -21035,7 +21035,7 @@ function Deref(Info: PPTypeInfo): PTypeInfo;
 
 function GetTypeData(const info: TTypeInfo): pointer; {$ifdef HASINLINE}inline;{$endif}
 begin
-  result := AlignTypeData(PAnsiChar(@info.Name[1])+ord(info.Name[0]));
+  result := AlignTypeData(pointer(@info)+2+PByte(pointer(@info)+1)^);

```

Comment:
I suspect `PByte(@info)`  could be Delphi compatible or perhaps `PAnsiChar(@info)+2+PByte(PAnsiChar(@info)+1)^`

Reply:
but I don't see any logical difference with the former `PAnsiChar(@info.Name[1])+ord(info.Name[0])` expression, which is used in several places of the unit